relaybot.py

```diff
@@ -52,7 +53,15 @@ def get(option):
             options["nickServPassword"] = get("nickServPassword")
 
         factory = factory(options)
-        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
+        optionAsBoolean = { "": False, "false": False, "no": False, "true": True, "yes": True }
+        sentinel = object()
+        ssl = options.get('ssl', sentinel)
+        if(sentinel == ssl):

```

Comment:
I'm used to `value == variable` checks just from C habit (makes accidental assignment less likely).
I prefer that to the explicit check but I don't especially mind one way or the other.